pkg/scheduler/factory/factory_test.go

```diff
@@ -264,7 +265,7 @@ func TestDefaultErrorFunc(t *testing.T) {
 		// whole error handling system in the future. The test will time
 		// out if something doesn't work.
 		time.Sleep(10 * time.Millisecond)
-		got, exists, _ := queue.Get(testPod)
+		exists, err := internalqueue.IsIdenticalErrorPodExists(queue, testPod)
```

Comment:
This doesn't look right to me. We should create a similar "Get" (or "Has") function for PQ.

Reply:
Thanks for the review. Yes. I supposed to add a "Get" for PQ after talked with you before. But I then found Get() is actually not an interface of SchedulingQueue. Instead it is a cache.FIFO's interface.

Someone implemented a local function getUnschedulablePod() in scheduling_queue_test.go to get pod from unschedulableQ. It looks to me the design of ScheduleQueue() interface may not want to expose a Get for the pod in queue. So I add this function to only compare the input pod with the one in PQ and only return the comparison results instead of the pod entity. Insights?

Reply:
> But I then found Get() is actually not an interface of SchedulingQueue.

As we're testing PQ here, not the ScheduleQueue interface, so it's fine to just create a public function for PQ.